components/bookstore.html

```diff
@@ -0,0 +1,226 @@
+                    <div class="badge-icon">1</div>
+                </a>
+                <button id="login-btn" class="login-btn-bookhub">Login</button>
+                <button id="signUp-btn" class="signUp-btn-bookhub">Sign up</button>
+            </div>
+        </div>
+    </header>
+    <!-- header section ends -->
+
+    <!-- Filter section -->
+    <main class="bookhub-container">
+        <section class="filter-drawer">
+            <div class="filter-header">
+                <h3>Filters</h3>
+                <button class="btn">Clear</button>
+            </div>
+            <span class="hr_line-book"></span>
+            <div class="filter-category filter">
+                <h4>Category</h4>
+                <label for="fiction">
+                    <input type="checkbox" id="fiction" value="1">Fiction
+                </label>
+                <label for="Non-fiction">
+                    <input type="checkbox" id="Non-fiction" value="2">Non-Fiction
+                </label>
+                <label for="Children & Teens">
+                    <input type="checkbox" id="Children & Teens" value="3">Children & Teens
+                </label>
+                <label for="Personal Development">
+                    <input type="checkbox" id="Personal Development" value="3">Personal Development
+                </label>
+            </div>
+            <span class="hr_line-book"></span>
+            <div class="filter-price filter">
+                <h4>Sort by</h4>
+                <label for="low-price">
+                    <input type="radio" id="low-price" value="low" name="price">Price - Low to High
+                </label>
+                <label for="high-price">
+                    <input type="radio" id="high-price" value="high" name="price">Price - High to Low
+                </label>
+            </div>
+            <span class="hr_line-book"></span>
+            <div class="filter-rating filter">
+                <h4>Rating</h4>
+                <label for="rating-4">
+                    <input type="radio" id="rating-4" value="4" name="rating">4 Stars & above
+                </label>
+                <label for="rating-3">
+                    <input type="radio" id="rating-3" value="3" name="rating">3 Stars & above
+                </label>
+                <label for="rating-2">
+                    <input type="radio" id="rating-2" value="2" name="rating">2 Stars & above
+                </label>
+                <label for="rating-1">
+                    <input type="radio" id="rating-1" value="1" name="rating">1 Stars & above
+                </label>
+            </div>
+        </section>
+        <!-- Card section starts -->
+        <section class="cards_for-book">
+
+            <div class="card-vertical flex-items">
+                <div class="image-container badge-container">
+                    <img class="img-responsive card-img" src="../assest/fiction-book2.jpg" alt="">
```

Comment:
Always give name to **alt**

Reply:
![Screenshot from 2022-02-13 18-03-38](https://user-images.githubusercontent.com/65344146/153753234-784872c1-cb91-4683-8a08-bdb184acd72e.png)
Why this extra search icon

Reply:
Yes, I do but I try it again👍🏻

Reply:
@anuj-kumary  Done👍🏻
Thank you for highliting